index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Holliday Planner</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="nav-bar">
+        <img src="./picture/Logo.png" alt="brand-logo" width="80px" height="auto">
+        <div class="nav-menu">
+        <nav class="nav-links">
+            <ul>
+                <li><a href="#">Home</a></li>
+                <li><a href="#">Find tours</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+        </nav>
+        </div>
+        </div>
+        <h1>Holiday Planner</h1>
+        <div class="banner">
+            <div class="card">
```

Comment:
try to respect the mock-up as much as you can: the distances between the cards are only a few pixels , checkout http://www.spadixbd.com/freetools/jruler.htm mentioned in the requirements https://confluence.fortech.ro/pages/viewpage.action?spaceKey=U2M78&title=Hands+on%3A+Holiday+planner+app